pkg/logstash.sysv

```diff
@@ -141,7 +141,6 @@ configtest() {
     return 1
   fi
 
-  JAVA_OPTS=${LS_JAVA_OPTS}
   HOME=${LS_HOME}
   export PATH HOME JAVA_OPTS LS_HEAP_SIZE LS_JAVA_OPTS LS_USE_GC_LOGGING
```

Comment:
> A user can supply JAVA_OPTS

if they're supplying it, aren't they supplying it from the environment so there's no need to do `export JAVA_OPTS` again? I think this is mostly a style-and-intent concern, not a functional problem, though.